flink-connectors/flink-connector-filesystem/src/test/java/org/apache/flink/streaming/connectors/fs/bucketing/BucketingSinkFaultToleranceITCase.java

```diff
@@ -144,6 +148,10 @@ public void postSubmit() throws Exception {
 
 		while (files.hasNext()) {
 			LocatedFileStatus file = files.next();
+//			if (!file.getPath().getName().startsWith(PART_PREFIX)) {
+				// ignore files that don't match with our expected part prefix
+//				continue;
+//			}
```

Comment:
commented code should be removed

Reply:
Ah, this is the code I actually added. I must have commented out by accident.